docs/guides/admin/docs/configuration/user-statistics.and.privacy.md

```diff
@@ -1,8 +1,6 @@
 User Statistics and Privacy
 ===========================
 
-*There exists a newer and more complete tracking service using Matomo included [as a module](../modules/player.matomo.tracking.md).*
```

Comment:
This file went away, but as far as I know Paella can/does use the old tracking tool.  If this is no longer true maybe this file should go away entirely?  Or be replaced with Paella specific docs?